panel/models/tabulator.ts

```diff
@@ -478,19 +482,14 @@ export class DataTabulatorView extends HTMLBoxView {
 
   tableBuilt(): void {
     this._building = false
-    this.setHidden()
```

Comment:
Is this removal related to the change further down (`columns.push({field: '_index', frozen: true, visible: false})`)?

Reply:
Yes, the aim is to configure as much as possible in the table configuration rather than setting the options after initialization.